src/fulfillment/docker-build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+docker build . -t mushop-fulfillment:latest
```

Comment:
convention is `mushop/<svc>` (even though we push to `ateam/mushop-svc` in OCIR). Also, do we need the shell script? The other option is `./ci/build.sh src/fulfillment` which will this already

Reply:
Will fix. The shell script is generated by the micronaut cli. Will remove. Not used really

Reply:
done